pasture-tools/src/bin/render/render.rs

```diff
@@ -0,0 +1,135 @@
+use std::path::{Path, PathBuf};
+use pasture_io::las::LASReader;
+use clap::{App, Arg};
+
+use winit::{
+    event::{Event, WindowEvent},
+    event_loop::{ControlFlow, EventLoop},
+    window::Window,
+};
+
+use nalgebra::Vector2;
+
+mod renderer;
+
+struct Args {
+    pub input_file: PathBuf,
+}
+
+fn get_args() -> Args {
+    let matches = App::new("pasture renderer")
+        .version("0.1")
+        .author("Jan Kelling <jan.kelling@stud.tu-darmstadt.de>")
+        .about("Renders point cloud files")
+        .arg(
+            Arg::with_name("INPUT")
+                .short("i")
+                .takes_value(true)
+                .value_name("INPUT")
+                .help("Input point cloud file")
+                .required(true),
+        )
+        .get_matches();
+
+    let input_file = PathBuf::from(matches.value_of("INPUT").unwrap());
+
+    Args {
+        input_file,
+    }
+}
+
+async fn run(event_loop: EventLoop<()>, window: Window, path: Option<PathBuf>) {
+    let mut renderer = renderer::Renderer::new(&window).await;
+
+    // For wasm: cannot load from file atm. Include statically.
+    let mut reader = if let Some(reader_path) = path {
+        LASReader::from_path(reader_path).expect("Failed to open lad file")
+    } else {
+        // let buf : &[u8] = include_bytes!("/home/jan/loads/red-rocks.laz");
+        let buf : &[u8] = include_bytes!("../../../../pasture-io/examples/in/10_points_format_1.las");
+        let c = std::io::Cursor::new(buf);
+        LASReader::from_read(c, true).expect("Failed to create LASReader")
```

Comment:
This should be `from_read(c, false)`, because you are including an uncompressed LAS file. If you put `true` here, it crashes because it is expecting an `LAZ` file instead